assam_budget_board/templates/grant_exp.html

```diff
@@ -105,6 +105,9 @@ <h5 class="summary-sec-indicator-title">{{ fiscal_year.actuals }} </h5>
 {% addtoblock "js" %}
 <script src="https://d3js.org/d3.v3.min.js"></script>
 <script src="{% static 'js/nv.d3.min.js' %}"></script>
+<script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
We can remove this, we already have included JQuery in the base template.

Reply:
okey